contracts/TapETH.sol

```diff
@@ -271,10 +273,8 @@ contract TapETH is Initializable, ITapETH {
   function setTotalSupply(uint256 _amount) external {
     require(pools[msg.sender], "TapETH: no pool");
     require(_amount != 0, "TapETH: no pool");
-    uint256 _deltaBuffer = Math.min(buffer, _amount);
+    uint256 _deltaBuffer = (buffer * _amount) / BUFFER_DENOMINATOR;
     _totalSupply += _amount - _deltaBuffer;
-    totalRewards += _amount;
```

Comment:
`totalRewards` is only used here, do you want to remove it?

Reply:
yes I thought that I have already removed

Reply:
I think we need the `totalRewards` to record the rewards. 
@ukby1234 What do you think?

Reply:
not sure though

Reply:
This is just a rewards record and has no logical processing. But I suggest leaving it